### DISCUSSION TITLE
Is it possible to get help with warnings with the new version 14.5.0?

### DISCUSSION THREAD

**Discussion by fixx-er** [hassio-addons/addon-node-red#1677] [Q&A]
Below is what is showing up under "debug":

9/14/2023, 8:23:47 AM
msg : string[270]
"Deprecated API warning: Calls to RED.util.evaluateJSONataExpression must include a callback. This will not be optional in Node-RED 4.0. Please identify the node from the following stack and check for an update on npm. If none is available, please notify the node author."
9/14/2023, 8:23:47 AM
msg : string[754]
"Error: ↵    at Object.evaluateJSONataExpression (/opt/node_modules/@node-red/util/lib/util.js:775:18)↵    at JSONataService.evaluate (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/common/services/JSONataService.js:58:39)↵    at ComparatorService.getComparatorResult (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/common/services/ComparatorService.js:108:94)↵    at PollState.getComparatorResult (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/nodes/BaseNode.js:128:39)↵    at PollState.onTimer (/opt/node_modules/node-red-contrib-home-assistant-websocket/dist/nodes/poll-state/controller.js:82:30)↵    at listOnTimeout (node:internal/timers:569:17)↵    at processTimers (node:internal/timers:512:7)"
9/14/2023, 8:24:46 AMnode: sum min
status : msg.payload : number
1

**Comment by frenck** [comment]
This is an issue with the `node-red-contrib-home-assistant-websocket`, which is used by this project; but not maintained or created here. Please report this with the project upstream.

..?Frenck
